docs/runtime/jvm-metrics.md

```diff
@@ -202,7 +207,19 @@ Note that this is the number of platform threads (as opposed to virtual threads)
 <!-- semconv metric.jvm.thread.count(full) -->
 | Attribute  | Type | Description  | Examples  | Requirement Level |
 |---|---|---|---|---|
-| [`thread.daemon`](../general/attributes.md) | boolean | Whether the thread is daemon or not. |  | Recommended |
+| `jvm.thread.daemon` | boolean | Whether the thread is daemon or not. |  | Recommended |
+| `jvm.thread.state` | string | State of the thread. | `runnable`; `blocked` | Recommended |
```

Comment:
I'm thinking if it would worth adding some kind of and indicator (footnote) for the thread state examples that shows where is the rest of that list in case there will be more, e.g.:

`runnable`; `blocked` [1]

[1]: `jvm.thread.state` MUST be one of the following:
...

Reply:
This part is code generated, I opened https://github.com/open-telemetry/build-tools/issues/223 to track 👍